eng/Build.props

```diff
@@ -62,6 +65,13 @@
     <ProjectToBuild Include="@(_NativeProjectToBuild->Exists())" />
   </ItemGroup>
 
+  <!-- Override ProjectToBuild items if specified -->
+  <ItemGroup Condition="'@(OverrideProjectToBuild)' != ''">
+    <ProjectToBuild Remove="@(ProjectToBuild)" />
+
+    <ProjectToBuild Include="@(OverrideProjectToBuild)" />
+  </ItemGroup>
```

Comment:
Why do it this way? Instead of just setting `<ProjectToBuild>` to the right thing from the start?